### ISSUE TITLE
Request: Static and single text input

### ISSUE THREAD

**Issue by kolban-google** [data-integrations/data-generator#4]
I'd like to see an augmentation where we generate data that is sourced from a textbox widget and results in only a single instance of a split where the output schema is a single record with a field of String type called body.  I'd like to be able to enter the string content of the record in the configuration properties of the plugin.

**Comment by albertshau** [comment]
makes sense. this could be done easily by adding a string constant generator.

**Comment by kolban-google** [comment]
Thank you my friend.  I have submitted a pull request for your considerations....

https://github.com/data-integrations/data-generator/pull/5

Very nice work in the internals.  You made it very easy for contributors to extend.  Beautifully elegant code.
